resources/lang/en/settings.php

```diff
@@ -1230,6 +1230,10 @@
             'description' => 'Route entries older than',
             'help' => 'Cleanup done by daily.sh',
         ],
+        'redis_sentinel_password' => [
```

Comment:
im not so sure this is applied into the python service? i might be wrong tho

Reply:
I think I just got a little overzealous trying to get things in docs, I'll remove this

Reply:
@banachtarski-91 The translation string is needed.  @Jellyfrog looking at the code it does pull it from the config.